dags/nocp/scripts/run_ocm_benchmark.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+export UUID=$(uuidgen | head -c8)-$AIRFLOW_CTX_TASK_ID-$(date '+%Y%m%d')
+echo "############################################"
+echo "# Benchmark UUID: ${UUID}"
+echo "############################################"
+
+env >> /tmp/environment.txt
+echo $UUID
+
+run_ocm_benchmark(){
+    echo "OCM Benchmark will be began.."
+    echo "Orchestration host --> $ORCHESTRATION_HOST"
+
+    git clone -q --depth=1 --single-branch --branch master https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git /tmp/perf-dept
+    export PUBLIC_KEY=/tmp/perf-dept/ssh_keys/id_rsa_pbench_ec2.pub
+    export PRIVATE_KEY=/tmp/perf-dept/ssh_keys/id_rsa_pbench_ec2
+    chmod 600 ${PRIVATE_KEY}
+
+    echo "Transfering the environment variables and OCM test config file to the orchestration host"
+    scp -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -i ${PRIVATE_KEY}  /tmp/environment.txt root@${ORCHESTRATION_HOST}:/tmp/environment_new.txt
+
+    echo "Running ocm-load-test inside $ORCHESTRATION_HOST"
+    ssh -t -o 'ServerAliveInterval=30' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -i ${PRIVATE_KEY} root@${ORCHESTRATION_HOST} << EOF
+
+    mkdir /tmp/${UUID}
+    git clone --single-branch --branch ci_test https://github.com/venkataanil/ocm-api-load /tmp/${UUID}
```

Comment:
I guess here we need to change to `cloud-bulldozer` repo

Reply:
Maybe a variable here? so we can test at some point different repos?

Reply:
Recently "multiple ocm accounts" support added in https://github.com/cloud-bulldozer/ocm-api-load/ however the corresponding the CI templates (https://github.com/cloud-bulldozer/ocm-api-load/tree/main/ci/templates ) are not updated.
So we are using ocm-api-load from my repo.